src/common/kyc/index.js

```diff
@@ -212,8 +223,8 @@ const loadRelyingPartyOperation = rpName => async (dispatch, getState) => {
 	const config = rp.relying_party_config;
 
 	try {
-		const session = mpService.createRelyingPartySession(config);
-		await session.establish();
+		const session = getSession(config, authenticate);
```

Comment:
@andregoncalves you might need to add an `await` here. cheers

Reply:
👍